run_api.py

```diff
@@ -23,5 +24,5 @@
     if len(sys.argv) > 1 and sys.argv[1].lower().startswith('test'):
         doctest.testmod()
     else:
-        debug = not os.getenv('PRODUCTION')
+        debug = not ast.literal_eval(os.getenv('PRODUCTION', False))
```

Comment:
But parsing environment variables to booleans is brittle however we do it--if a developer sets `PRODUCTION` to `"false"` instead of `"False"`, they'll get a potentially confusing error. What about renaming `PRODUCTION` to something like `FEC_ENVIRONMENT`, with values of `"dev"`, `"staging"`, or `"production"`? Then if a developer has an invalid value, we can default to `"dev"`.